packages/eas-cli/src/project/publish.ts

```diff
@@ -295,4 +302,5 @@ export async function uploadAssetsAsync(assetsForUpdateInfoGroup: CollectedAsset
       throw new Error('Asset upload timed out. Please try again.');
     }
   }
+  return uniqueUploadedAssetCount;
```

Comment:
I think it would be useful to return three values: the total number of assets (assets.length), the number of unique assets (uniqueAssets.length), and the number of newly uploaded assets (uniqueUploadedAssetCount). This way we can show a more detailed breakdown.

Reply:
I just updated the command to update the spinner text in real-time as Kim suggested. So it's no longer necessary to return the additional two values from this function, but I left them in the object because it doesn't hurt 🤷